.changeset/seven-carpets-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/world": patch
+---
+
+Require that namespaces exist before transferring ETH to them.
```

Comment:
```suggestion
Namespace balances can no longer be transferred to non-existent namespaces.
```
reworded for consistent tense with other changesets and to be a bit more specific about namespace balances